llvm/include/llvm/Object/XCOFFObjectFile.h

```diff
@@ -787,6 +794,14 @@ class XCOFFSymbolRef {
 
   uint64_t getValue64() const { return Entry64->Value; }
 
+  const XCOFFObjectFile *getObject() const {
+    return cast<XCOFFObjectFile>(BasicSymbolRef::getObject());
+  }
```

Comment:
Does this actually need to be a public method? Does it need to be a separate method at all in fact?

Reply:
Done. Since for now, there is only one user for this function, I inlined this function to its caller.